.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Contribuição
+
+## 📝 Descrição
```

Comment:
Fala @buguno, você realmente é o cara!! Estou montando justamente uma apresentação para nós ajustarmos boas praticas de contribuição e codeview na BW e vc já tá na vibe 🎉🎉🎉🎉🎉... parabéns!

Maninho eu só tem algumas observaçoes com relação o seu PR, o que voce acha de adicionarmos alguns exemplos e uns campos novos:

## 📝 Descrição

Descreva com detalhes do que se trata o PR. EX:

Criação de uma API para criação do usuário utilizando Firebase.... etc...

## 🧐Motivo e Contexto? 

Descrição do porque o PR é necessario e motivos pelo quais novas bibliotecas foram implementadas. 

Ex:
Essa feature é necessaria pois ela será o ponto de entrada para o registro do usuário na platforma da BW.

//OU para Bugs

Devido a propriedade X não estar sendo validada no endpoint de signtup do usuário, o backend estava retornando SERVER INTERNAL ERROR para o usuário final.

```golang
// Snippet from code where the bug is happen
type User struct {
	name string
	gender string
	phone string // <-- This required field was not been validated
}
```

### 📚Novas libs:

firebase-sdk: Realizamos uma busca para escolher a melhor API para Singup de usuário na internet, e aparentemente o firebase utiliza sua propria bliblioteca para realizar esse trabalho, veja aqui o link do spike the escolha de serviço <LINK PARA O DOCUMENTO>

fiber: Para contruir a nossa camada de web-framework, escolhe-mos essa biblioteca pois aparentemente ela é a mais rapida do mercado e: x, y e z


## ✅ Como? / ✅ Checklist

- [ ] Adicona(ei) firebase lib no projeto
- [ ] Manda(ei) o token do firebase para o backend para finalizar o registro do usuario.
- [ ] Adiciona(ei) lint no projeto

## 🧪 Como testar

Descreva como testar as funcionalidades ou correções, forneça instruções de como rodar os testes e liste todos os detalhes relevantes.

## 📺 Screenshots & demo

Mostre Screenshots, videos ou gifs da funcionalidade rodando no seu ambiente de desenvolvimento si necessario.

Reply:
Ótimo @mendesbarreto essa semana podemos dar os toque finais e fazer as alterações no arquivo.